features/registerHealthCertificate.feature

```diff
@@ -0,0 +1,31 @@
+@RegistryService
```

Comment:
See comment below regarding tag - would add @RegisterHealthCertificate to the feature and change existing @RegisterHealthCertificate to something like @RegisterHealthCertificate-valid.

Reply:
Same as above, the feature file needs the @RegistryService to set the endpoint. You can use more than one tag in Gherkin.